src/index.js

```diff
@@ -1,6 +1,11 @@
 import './index.scss';
 import App from './App.js';
+// import Api from '../utils/Api/Api.js';
 
 // const Root = document.querySelector('#root');
+// const ap = new Api();
+// (async () => {
+//   console.log(await ap.get());
+// })();
```

Comment:
- [ ] Please remove all commented code as it makes your work look untidy 🙏

Reply:
@codepantha Well later I need this code but still for the review I removed my unused code.